packages/scripts/src/create-webpack-configs.ts

```diff
@@ -43,27 +44,39 @@ export function createWebpackConfigs(options: ICreateWebpackConfigsOptions): web
         publicPath = '',
         createWebpackConfig,
         environments: { electronRendererEnvs, webEnvs, workerEnvs },
+        context,
     } = options;
 
     if (!baseConfig.output) {
         baseConfig.output = {};
     }
     baseConfig.output.publicPath = publicPath;
+
+    const packageJsonPath = fs.findClosestFileSync(__dirname, 'package.json'); // We want "our" package to extract the package name;
+    if (!packageJsonPath) {
+        // Should never happen
+        throw new Error('Cannot find package.json for determening cache dir');
+    }
+    const packageJson = fs.readJsonFileSync(packageJsonPath) as { name: string };
+
+    const projectCacheDir = findCacheDir({ name: packageJson.name, cwd: context, create: true });
+    if (!projectCacheDir) {
+        // We cannot write to the cache dir for some reason, usually access we just throw
+        throw new Error('Please make sure you have a package.json file and have right access to node modules');
+    }
     const configurations: webpack.Configuration[] = [];
-    const virtualModules: Record<string, string> = {};
 
     if (webEnvs.size) {
-        const plugins: webpack.WebpackPluginInstance[] = [new VirtualModulesPlugin(virtualModules)];
         const entry: webpack.Entry = {};
         configurations.push(
             createWebpackConfig({
                 ...options,
                 baseConfig,
                 enviroments: webEnvs,
                 target: 'web',
-                virtualModules,
-                plugins,
+                plugins: [],
```

Comment:
if we don't really use this field anymore (plugins), remove it from the options of createWebpackConfig

Reply:
Done